src/CoreWCF.ConfigurationManager/src/ConfigurationManagerExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
There shouldn't be any source files at the root of the project. Can you move the collection of source files to the correct folder?